play.pokemonshowdown.com/src/battle-animations.ts

```diff
@@ -2473,8 +2473,42 @@ export class PokemonSprite extends Sprite {
 			});
 		}
 	}
-	animTransform(pokemon: Pokemon, isCustomAnim?: boolean, isPermanent?: boolean) {
-		if (!this.scene.animating && !isPermanent) return;
+	customAnimAndIfCry(speciesid: ID, isDynamaxing?: boolean): boolean {
+		console.log("customAnimAndIfCry is running");
+		const scene = this.scene;
+		if (isDynamaxing) {
+			BattleOtherAnims.megaevo.anim(scene, [this]);
```

Comment:
i saw that there was a dynamax anim, but it wasn't previously being used.  It would be easy to enable it now.  But maybe thats a follow up.